examples/react/trimmed/src/components/shadcn-table.tsx

```diff
@@ -50,6 +50,14 @@ export function ShadcnTable<TData extends RowData, TValue>(props: {
 }) {
   const { table, columns } = props;
 
+  // Helper to safely access metadata handlers
+  const meta = table.options.meta as
```

Comment:
This needs to be tweaked. We can probably add this on to the `meta` of the `row`, or possibly even implement it as a ["Custom Feature"](https://tanstack.com/table/latest/docs/guide/custom-features) in TanStack Table.